propnet/core/tests/test_mp_interface.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from propnet.core.symbols import *
+from propnet.symbols import *
+from propnet.ext.matproj import *
+
+class MPInterfaceTest(unittest.TestCase):
+    """ """
```

Comment:
Empty docstring. Either add a docstring or delete. I don't usually see much point in adding a docstring if the test is self-explanatory, but may be helpful

Reply:
Cool let's delete. PyCharm wasn't happy with me unless I put the quotes.

Reply:
I'll remove all empty docstrings